Description: skip UnixSocketTest on hurd, as unix sockets with SO_REUSEADDR are not supported there
 A little discussion about unix sockets with SO_REUSEADDR can be found on
 https://lists.gnu.org/archive/html/bug-hurd/2016-01/msg00039.html
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: no
Last-Update: 2016-04-28

--- a/tornado/test/httpserver_test.py
+++ b/tornado/test/httpserver_test.py
@@ -534,6 +534,8 @@
 
 @unittest.skipIf(not hasattr(socket, 'AF_UNIX') or sys.platform == 'cygwin',
                  "unix sockets not supported on this platform")
+@unittest.skipIf(sys.platform == 'gnu0',
+                 "unix sockets with SO_REUSEADDR not supported on this platform")
 class UnixSocketTest(AsyncTestCase):
     """HTTPServers can listen on Unix sockets too.
 
